### PR TITLE
Remove without_storage_info for the poa pallet

### PR DESCRIPTION
**Pull request by rajesh-nodle**
@Team Ignore this PR, Tracking History is too High, Recreating new one

Part of [#173](https://github.com/NodleCode/chain-workspace/issues/173)

**HighLights**

- Storage `Validators` is moved to bounded vector.
- Upperlimit  for  `Oracles`  is Configured via `MaxValidators`.
- Introduced new events `ValidatorsUpdated(u32)` & `ValidatorsMaxOverflow(u32, u32)`
- UnitTest Extension::Done
- Storage Migration (try_runtime)

**TODO**

**Ready for final Intake Review**

### CODE REVIEW

File: pallets/poa/Cargo.toml [hunk 1]
```diff
@@ -28,6 +32,9 @@ pallet-session = { git = "https://github.com/paritytech/substrate", default-feat
 sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
 sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
+sp-staking = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
```
Comment (simonsso) [inline pallets/poa/Cargo.toml:35]: Did we already decide to use the staking from upstream?

Comment (rajesh-nodle) [reply pallets/poa/Cargo.toml:35]: To import `SessionIndex` is defined as part of `sp-staking`

image

**Review by simonsso — CHANGES_REQUESTED** [review]
`cargo test` is also failing, is every file checked in?

**Comment by rajesh-nodle** [comment]
@Team Ignore this PR, Tracking History is too High, Recreating new one.

**Comment by rajesh-nodle** [comment]
Replaced with https://github.com/NodleCode/chain/pull/633

**Pull request closed without merging**